tests/php/Forms/FormTest.php

```diff
@@ -1364,7 +1364,7 @@ public function testValidateFormFields(array $values, bool $isValid)
         }
         $form = new Form(null, 'testForm', $fieldList, new FieldList([/* no actions */]));
 
-        $result = $form->validationResult();
+        $result = $form->validate();
```

Comment:
A couple of unit tests still called validationResult() which were missed in the CMS 5 PRs

Reply:
Should these be resolved in a new CMS 5 PR then?

Reply:
Probably should, will do

Reply:
Actually upon further inspection no they shouldn't

https://github.com/silverstripe/silverstripe-framework/commit/283d69a58188e2c23aac02768f2bbf4b56eb1bc6#diff-b26e65943a8d45b13830a9dc8b0a8471b23747fc12bf278afa7d484efcb33e7eR1367

Was introduced by a PR targetting 6.0, code isn't present in 5

https://github.com/silverstripe/silverstripe-framework/blame/5/tests/php/Forms/OptionsetFieldTest.php#L64

Code was changed to validate() in CMS 5 PR, looks was changed validationResult() in 6.0 as part of resolving a merge-up